column.php

```diff
@@ -668,9 +668,24 @@ private function sanitize_relationships( $relationships = array() ) {
 	 * @return string|null
 	 */
 	private function sanitize_default( $default = '' ) {
-		return is_null( $default )
-			? null
-			: wp_kses_data( $default );
+
+		// Null
+		if ( ( true === $this->allow_null ) && is_null( $default ) ) {
+			return null;
+
+		// String
+		} elseif ( is_string( $default ) ) {
+			return wp_kses_data( $default );
+
+		// Integer
+		} elseif ( $this->is_numeric( $default ) ) {
```

Comment:
```suggestion
		} elseif ( $this->is_numeric() ) {
```

`is_numeric()` doesn't have any parameters.

Reply:
Resolved in https://github.com/berlindb/core/pull/78/commits/aaa5be125180dc82a8de91eee062d4309ffb9fd9.